platforms/common/src/kernels/customCentroidBond.cc

```diff
@@ -3,14 +3,15 @@
  */
 KERNEL void computeGroupCenters(int numParticleGroups, GLOBAL const real4* RESTRICT posq, GLOBAL const int* RESTRICT groupParticles,
         GLOBAL const real* RESTRICT groupWeights, GLOBAL const int* RESTRICT groupOffsets, GLOBAL real4* RESTRICT centerPositions) {
-    LOCAL volatile real3 temp[64];
+    //
+    LOCAL volatile real3 temp[BLOCK_SIZE];
     for (int group = GROUP_ID; group < numParticleGroups; group += NUM_GROUPS) {
         // The threads in this block work together to compute the center one group.
 
         int firstIndex = groupOffsets[group];
         int lastIndex = groupOffsets[group+1];
         real3 center = make_real3(0);
-        for (int index = LOCAL_ID; index < lastIndex-firstIndex; index += LOCAL_SIZE) {
+        for (int index = LOCAL_ID; index < lastIndex-firstIndex; index += BLOCK_SIZE) {
```

Comment:
How is `BLOCK_SIZE` different from `LOCAL_SIZE`?

In other kernels where we need to define a macro for the thread block size, we call it `THREAD_BLOCK_SIZE` to be clear.